README.md

```diff
@@ -1,6 +1,6 @@
-# Data processing package (provisional name)
+# PBDP PyBamm Data Processing / Python Battery Data Processing
```

Comment:
```suggestion
# PBDP Python Battery Data Processing
```

Reply:
For now let's just use the more generic way, as the package should work completely independently from PyBaMM